pipeline/post-results.groovy

```diff
@@ -66,13 +66,14 @@ node(nodeName) {
 
             def metaData = readYaml file: "${tmpDir}/${resultDir}/metadata.yaml"
             def copyFiles = "cp -a ${tmpDir}/${resultDir}/results ${rpPreprocDir}/payload/"
+            def copyAttachments = "cp -a ${tmpDir}/${resultDir}/attachments ${rpPreprocDir}/attachments/"
```

Comment:
It is sufficient to do `def copyAttachments = "cp -a ${tmpDir}/${resultDir}/attachments ${rpPreprocDir}/`